src/Event/HttpFlowEvent.php

```diff
@@ -45,8 +45,8 @@ class HttpFlowEvent extends Event
     public function __construct($name, $request, $response)
     {
         $this->setName($name);
-        $this->response = $response;
-        $this->request = $request;
+        $this->setRequest($request);
+        $this->setResponse($response);
```

Comment:
refactored, use setter for `request` and `response` like the `setName()` one, so it covered in tests.